mapsforge-map/src/main/java/org/mapsforge/map/layer/hills/DemFileZipEntryFS.java

```diff
@@ -23,31 +24,36 @@
 
 public class DemFileZipEntryFS implements DemFile {
 
-    protected final ZipFile zipFile;
-    protected final ZipEntry zipEntry;
+    protected final File mFile;
+    protected final String mZipEntryName;
+    protected final long mZipEntrySize;
```

Comment:
@Sublimis Please do not use the `m` field naming, as we do not use it in Mapsforge.

It also overloads the commit history with unrelated changes in the pull request.